scrapy/squeues.py

```diff
@@ -25,9 +25,10 @@ def pop(self):
 def _pickle_serialize(obj):
     try:
         return pickle.dumps(obj, protocol=2)
-    # Python>=3.5 raises AttributeError here while
-    # Python<=3.4 raises pickle.PicklingError
-    except (pickle.PicklingError, AttributeError) as e:
+    # Python<=3.4 raises pickle.PicklingError here while
+    # Python>=3.5 raises AttributeError and
```

Comment:
does it mean 3.5 <= Python version < 3.6?

Reply:
Yes, let me clarify that. To be honest I have not seen any `AttributeError`, I just left what the previous comment said, but I phrased my addition poorly.

Reply:
Done, thanks for noticing!